apps/benchmarking-image-features/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+INCLUDE(../../cmakemodules/AssureCMakeRootFile.cmake)
+INCLUDE(../../cmakemodules/script_qt.cmake)
+
```

Comment:
Please check (I'm on the phone right now....), but I think you don't need/must include the script_qt here, since it's already included in the root cmake.

Reply:
sure I will check this and remove if not required